pkg/cmd/grafana-server/commands/diagnostics.go

```diff
@@ -12,24 +12,30 @@ import (
 )
 
 const (
-	profilingEnabledEnvName = "GF_DIAGNOSTICS_PROFILING_ENABLED"
-	profilingAddrEnvName    = "GF_DIAGNOSTICS_PROFILING_ADDR"
-	profilingPortEnvName    = "GF_DIAGNOSTICS_PROFILING_PORT"
-	tracingEnabledEnvName   = "GF_DIAGNOSTICS_TRACING_ENABLED"
-	tracingFileEnvName      = "GF_DIAGNOSTICS_TRACING_FILE"
+	profilingEnabledEnvName   = "GF_DIAGNOSTICS_PROFILING_ENABLED"
+	profilingAddrEnvName      = "GF_DIAGNOSTICS_PROFILING_ADDR"
+	profilingPortEnvName      = "GF_DIAGNOSTICS_PROFILING_PORT"
+	profilingBlockRateEnvName = "GF_DIAGNOSTICS_PROFILING_BLOCK_RATE"
+	profilingMutexRateEnvName = "GF_DIAGNOSTICS_PROFILING_MUTEX_RATE"
+	tracingEnabledEnvName     = "GF_DIAGNOSTICS_TRACING_ENABLED"
+	tracingFileEnvName        = "GF_DIAGNOSTICS_TRACING_FILE"
 )
 
 type profilingDiagnostics struct {
-	enabled bool
-	addr    string
-	port    uint64
+	enabled   bool
+	addr      string
+	port      uint64
+	blockRate uint64
+	mutexRate uint64
```

Comment:
these are both `int`s when called - negative values have meaning - so why not just set these to `int`?

```suggestion
	blockRate int
	mutexRate int
```

Reply:
After consideration I'm changing to `int` everywhere thanks 😄